### PR TITLE
Data modality 1 - Gyroscope

### PR DESCRIPTION
**Pull request by legoeruro**

### CODE REVIEW

File: unity/Assets/Scenes/MockData.unity [hunk 1]
```diff
@@ -1607,7 +1607,7 @@ MonoBehaviour:
   m_OnCullStateChanged:
     m_PersistentCalls:
       m_Calls: []
-  m_text: "localhost:8500\u200B\u200B"
+  m_text: "localhost:8500\u200B"
```
Comment (YiqinZhao) [inline unity/Assets/Scenes/MockData.unity:1610]: Hmmm, I wonder what is the \u200B thing?

File: unity/Assets/Scripts/ARFlow/ARFlowClientManager.cs [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+                }
+                if (_activatedDataModalities["CameraDepth"])
+                {
+                    var CameraDepth = new RegisterRequest.Types.CameraDepth()
+                    {
+                        Enabled = true,
+#if UNITY_ANDROID
+                        DataType = "u16", // f32 for iOS, u16 for Android
+#endif
+#if UNITY_IPHONE
```
Comment (YiqinZhao) [inline unity/Assets/Scripts/ARFlow/ARFlowClientManager.cs:132]: Deprecated. Use UNITY_IOS instead. Check: https://docs.unity3d.com/Manual/PlatformDependentCompilation.html

File: python/examples/minimal/minimal.py [hunk 3]
```diff
@@ -9,9 +9,24 @@
 
 import arflow
 
+import rerun as rr
```
Comment (YiqinZhao) [inline python/examples/minimal/minimal.py:12]: Should we use `rr` or `self.recorder`?

**Review by YiqinZhao — COMMENTED** [review]
Added comments. Looks like there are some code for other modalities. Do you want to merge this PR now for just adding support for the gyroscope? Or wait until you push more code for other modalities?

File: unity/Assets/Scenes/MockData.unity (hunk 1 above)
Comment (legoeruro) [reply unity/Assets/Scenes/MockData.unity:1610]: I think this is a hidden character; with this I couldn't press connect (bugged me out for a few days) so deleting it resolved the issue. I think unity auto-added the character to the end of a TMP text input box

File: unity/Assets/Scripts/ARFlow/ARFlowClientManager.cs (hunk 2 above)
Comment (legoeruro) [reply unity/Assets/Scripts/ARFlow/ARFlowClientManager.cs:132]: Good catch, thank you for the note.

File: python/examples/minimal/minimal.py (hunk 3 above)
Comment (legoeruro) [reply python/examples/minimal/minimal.py:12]: Ah yeah we should use self.recorder; that was for testing a new API in the new rerun version which I've pushed a commit to remove the testing.

**Review by YiqinZhao — APPROVED** [review]
LGTM.

**Pull request merged**